crates/noirc_frontend/src/hir/resolution/import.rs

```diff
@@ -152,6 +152,7 @@ fn resolve_name_in_module(
             ModuleDefId::FunctionId(_) => panic!("functions cannot be in the type namespace"),
             // TODO: If impls are ever implemented, types can be used in a path
             ModuleDefId::TypeId(id) => id.0,
+            ModuleDefId::TypeAliasId(_) => unimplemented!(),
             ModuleDefId::GlobalId(_) => panic!("globals cannot be in the type namespace"),
         };
```

Comment:
I suppose its fine to panic here for now and leave this as a later PR. I should make an issue anyway to turn these into actual user-facing errors rather than panics.